tests/installation/disk_activation.pm

```diff
@@ -23,24 +23,35 @@ sub format_dasd {
     }
 }
 
+sub add_zfcp_disk {
+    my $channel = shift;
+    assert_screen 'zfcp-disk-management';
+    send_key 'alt-a';
+    assert_screen 'zfcp-add-device';
+    send_key 'alt-a';
+    assert_screen 'zfcp-channel-select';
+    for (1 .. 8) { send_key "backspace"; }
+    type_string "$channel";
+    assert_screen "zfcp-channel-$channel-selected";
+
+    send_key $cmd{next};
+
+    # use allow_lun_scan
+    # popup was removed for sle15, allow_lun_scan still works though
+    if (!is_sle '15+') {
```

Comment:
Now we can write is_sle('<15') to do these checks

Reply:
changed